exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/registry/RemoteFunctionRegistry.java

```diff
@@ -189,6 +188,7 @@ private void prepareStores(PersistentStoreProvider storeProvider, ClusterCoordin
    * if not set, uses user home directory instead.
    */
   private void prepareAreas(DrillConfig config) {
+    logger.info("Preparing three remote udf areas: staging, registry and tmp.");
```

Comment:
Although we suggested keeping the file path stuff as-is, the new logging messages are great; let's keep them.